cmd/internal/cli/actions.go

```diff
@@ -172,7 +174,9 @@ var ExecCmd = &cobra.Command{
 			execVM(cmd, args[0], a)
 			return
 		}
-		execStarter(cmd, args[0], a, "")
+		imageArg := os.Getenv("IMAGE_ARG")
```

Comment:
Could you please move the redundant code : 
```
   imageArg := os.Getenv("IMAGE_ARG")
   os.Unsetenv("IMAGE_ARG")
```
in `execStarter` to have something like :
```
func execStarter(cobraCmd *cobra.Command, image string, args []string, name string) {
    imageArg := os.Getenv("IMAGE_ARG")
    os.Unsetenv("IMAGE_ARG")
    ...
}
```

Reply:
I made these adjustments to make things less redundant